core/src/main/java/com/linecorp/armeria/internal/metric/RequestMetricSupport.java

```diff
@@ -200,4 +236,43 @@ public Timer totalDuration() {
             return totalDuration;
         }
     }
+
+    private static class DefaultClientRequestMetrics
+            extends AbstractRequestMetrics implements ClientRequestMetrics {
+
+        private final Counter writeTimeout;
+        private final Counter responseTimeout;
+
+        DefaultClientRequestMetrics(MeterRegistry parent, MeterIdPrefix idPrefix) {
+            super(parent, idPrefix);
+            writeTimeout = parent.counter(idPrefix.name("writeTimeout"), idPrefix.tags());
+            responseTimeout = parent.counter(idPrefix.name("responseTimeout"), idPrefix.tags());
```

Comment:
- `writeTimeouts` and `responseTimeouts`
- or shoule we name them `timeouts` and add a tag? e.g. `timeouts{cause=WriteTimeoutException}` and `timeouts{cause=ResponseTimeoutException}`

Reply:
Great idea to name it as `timeouts` and add a tag. Fixed, thank you!